### PR TITLE
Adjustments for true plugin nature of China plugin

### LINKED ISSUE #772: Adjust China plugin page to mention true plugin nature
The China Plugin dependency structure is being adjusted so that it's a "true" plugin. It will no longer include the Maps SDK. The Maps SDK dependency will now have to be explicitly declared, which means there's now greater flexibility for testing and using various versions of the Mapbox Maps SDK for Android.

Code and writing on https://www.mapbox.com/android-docs/plugins/overview/china should be adjusted accordingly.

cc @Chaoba @chriswu42 @springmeyer

### PR DESCRIPTION
**Pull request by langsmith**
Resolves #772 by updating the writing/code on https://www.mapbox.com/android-docs/plugins/overview/china/ to account for the China Plugin's new structure. The plugin is now more of a true plugin because it no longer includes the Maps SDK. 

@electrostat [recently updated the core documentation code snippets](https://github.com/mapbox/android-docs/pull/762/files), so this pr updates the location-related code which is present in the `Shift Location` section of https://www.mapbox.com/android-docs/plugins/overview/china/

cc @chriswu42 @springmeyer @Chaoba

### CODE REVIEW

**Comment by langsmith** [comment]
screencapture-localhost-8080-android-docs-plugins-overview-china-2018-12-13-11_16_52

File: src/pages/plugins/overview/china.md [hunk 1]
```diff
@@ -6,32 +6,46 @@ prependJs:
...
 ---
 
-Traditional map services are either blocked in China or suffer from slow internet connections. Our mapbox.cn infrastructure allows for unparalleled speed advantages for anyone using our maps in China or through Chinese mobile carriers internationally. The Mapbox China Plugin for Android specifically bundles the Mapbox Maps SDK for Android and automatically configures the Maps SDK to ensure that the correct endpoints are being called. Accurate endpoints ensure that a mobile device retrieves the correct map tiles, map styles, and other location information. Additionally, the plugin allows for annotation shifting, which ensures all your overlays and annotations are accurately depicted on the map.
+Traditional map services are either blocked in China or suffer from slow internet connections. Our mapbox.cn infrastructure allows for unparalleled speed advantages for anyone using our maps in China or through Chinese mobile carriers internationally. The Mapbox China Plugin for Android is built on top of the Mapbox Maps SDK for Android and provides the necessary tools for developers to build China-related mobile experiences with location data.
```
Comment (colleenmcginnis) [inline src/pages/plugins/overview/china.md]: What does `China-related` mean? Is that just short for `using our maps in China or through Chinese mobile carriers internationally`?

File: src/pages/plugins/overview/china.md [hunk 2]
```diff
@@ -48,7 +62,7 @@ Another option is to drop the architectures you do not need to support for your
...
+The plugin has wrapper classes to be used in place of their "typical" counterparts. For example, in the Maps SDK for Android, you'd use the `MapView` inside your activities' layouts. When using this plugin, a lint error will appear telling you to use the `ChinaMapView` instead, which wraps the `MapView` object. The `ChinaMapView` class does nothing more but set default values for optimal performance inside China. This includes requesting map tiles from our Chinese servers. In the provided chart below, you'll find a list of the objects which you'd typically use in the Maps SDK for Android and their counterparts found inside of this plugin.
```
Comment (colleenmcginnis) [inline src/pages/plugins/overview/china.md]: >In the ~~provided~~ chart below, you'll find a list of the objects ~~which~~ you'd typically use in the Maps SDK for Android and their counterparts found inside of this plugin.

Comment (langsmith) [reply src/pages/plugins/overview/china.md]: Fixed

File: src/pages/plugins/overview/china.md (hunk 1 above)
Comment (langsmith) [reply src/pages/plugins/overview/china.md]: `China-related` is 👎 . Removed that sentence entirely. That paragraph is now:


>Traditional map services are either blocked in China or suffer from slow internet connections. Our mapbox.cn infrastructure allows for unparalleled speed advantages for anyone using our maps in China or through Chinese mobile carriers internationally. The Mapbox China Plugin for Android is built on top of the Mapbox Maps SDK for Android. The plugin automatically configures the Maps SDK to ensure that the correct Mapbox API endpoints are being called. Accurate endpoints ensure that a mobile device retrieves the correct map tiles, map styles, and other location information. Additionally, the plugin handles shifting of various GeoJSON geometries (polygons, lines, points, etc.), which ensures that data is accurately placed on the map.

**Pull request merged**